science/2022-03-03-science-blog-5/index.md

```diff
@@ -478,7 +478,7 @@ In the following figure, we compare the download numbers and the estimations of
 
 ## 5 Key figures from the CWA backend
 
-A further option for estimating the active apps/devices (in the narrower sense) are the [files of the daily keys](http://svc90.main.px.t-online.de/version/v1/diagnosis-keys/country/DE/date) downloaded by the users of the CWA and provided via the CWA backend. Even though the CWA accesses the shared daily keys, which are generated several times each day, the downloading of the daily packages takes place at most once per day. In the following, we analyse the data provided by [T-Systems International (TSI)](https://www.t-systems.com/) on January 26, 2022, which covers 46 days (from December 4, 2021, to January 19, 2022) and which is broken down by operating system. From this, we obtain 27.9 million active devices or 24.9 million active users (on January 19, 2022).
+<p>A further option for estimating the active apps/devices (in the narrower sense) are the <a href='http://svc90.main.px.t-online.de/version/v1/diagnosis-keys/country/DE/date' target='_blank' rel='noopener noreferrer'>files of the daily keys</a><span class='aside'>The resource that was available through this URL was removed when the Corona-Warn-App development ended on 5/31/2023.</span> downloaded by the users of the CWA and provided via the CWA backend. Even though the CWA accesses the shared daily keys, which are generated several times each day, the downloading of the daily packages takes place at most once per day. In the following, we analyse the data provided by <a href='https://www.t-systems.com/' target='_blank' rel='noopener noreferrer'>T-Systems International (TSI)</a> on January 26, 2022, which covers 46 days (from December 4, 2021, to January 19, 2022) and which is broken down by operating system. From this, we obtain 27.9 million active devices or 24.9 million active users (on January 19, 2022).</p>
```

Comment:
Write date as
"May 31, 2023" instead of
"5/31/2023"

Reply:
Thanks @MikeMcC399, we will remove the actual date, since the resource was removed a couple of days earlier.

Reply:
@dsarkar 
You're right!
Monitoring from @Ein-Tim shows that it became unavailable sometime during May 26, 2023. 🙂 

https://stats.uptimerobot.com/gKg1ZCqKJv